### DISCUSSION TITLE
Submitted Proof, instantly got rejected

### DISCUSSION THREAD

**Discussion by Infisreal** [community/community#18819] [GitHub Education]
I have my school email included, I provided the needed proof, and got rejected 17 times instantly. What should I do?
![image](https://user-images.githubusercontent.com/73630497/174396807-a9bdc417-7653-45f7-ae4f-5ccf52d7f6d6.png)

**Comment by Lumpish-Haggard** [comment]
Thats.....Hmm!
I think that a lot of different things are assessed before granting Github Student Developer's Pack as there are many people who try to spoof their way into it.
From using your Realtime Location through a browser (which bypasses most VPNs) to checking the META of the uploaded document to see if they are edited. The scrutiny increases the more time you are rejected.

Just upload a document/ID which CLEARLY mentions the date of your admission and the tenure of your course. (It helps to confirm that you are CURRENTLY a student and not just using your OLD student edu mail.)
Be warned that letters of ACCEPTANCE from Universities are many times rejected as they are not proof of ENROLLMENT.
